### ISSUE TITLE
Question:  Does this support JavaScript and TypeScript?

### ISSUE THREAD

**Issue by awentzel** [ShiftLeftSecurity/scan-docs#13]

**Comment by prabhu** [comment]
@awentzel scan bundles nodejsscan which supports javascript and nodejs. Please feel free to try on a repo and let us know your thoughts.

**Comment by awentzel** [comment]
@prabhu I'm new to security scanning. Is there documentation around best practices for what we should be scanning?  source, app, dist, etc?  I did a quick lookup of "nodejsscan" and got the impression it's best used for `dist` which are intended to deploy?  Thanks again.

**Comment by prabhu** [comment]
I just tried scanning fast-dna repo. Only the dependency scanner identified few issues. The SAST scanner didn't pick up any issue due to the known limitations. I will share a PR later to add this scan.
Regarding best practices, inspect uses a technology called [Code Property Graph](https://www.shiftleft.io/technology/). We build this from the raw source code of both code and dependencies. Once the graph is built, the inspect analyzers no longer requires the source code and can perform analysis in the cloud.

Scan product, in contrast, runs on the CI pipeline itself. It can automatically pick the right folders and files it needs and is built to work out-of-the-box without requiring any configuration. I have added a section on the documentation for [language types](https://slscan.io/en/latest/getting-started/#language-specific-scans) but as you might notice there is no special configuration required per type.
